site/content/contribute/more-info/focalboard/_index.md

```diff
@@ -12,11 +12,11 @@ Here's the process for contributing to Focalboard:
+2. Find [help wanted tickets that are up for grabs in GitHub](https://github.com/mattermost/focalboard/issues?q=is%3Aopen+is%3Aissue+label%3A%22Up+for+grabs%22+label%3A%22Help+Wanted%22). Comment to let everyone know you’re working on it and let a core contributor assign the issue to you. If there’s no ticket for what you want to work on, read about [contributions without a ticket]({{< ref "/contribute/more-info/getting-started/contributions-without-ticket" >}}).
 
-3. When your changes are checked in to your fork, follow the steps on our [contribution checklist]({{< ref "/contribute/getting-started/contribution-checklist" >}}). If this will be your first contribution, there is a standard [CLA](https://www.mattermost.org/mattermost-contributor-agreement/) that you will need to sign as part of this checklist.
+3. When your changes are checked in to your fork, follow the steps on our [contribution checklist]({{< ref "/contribute/more-info/getting-started/contribution-checklist" >}}). If this will be your first contribution, there is a standard [CLA](https://www.mattermost.org/mattermost-contributor-agreement/) that you will need to sign as part of this checklist.
 
-4. Submit your pull request for a [code review]({{< ref "/contribute/getting-started/code-review#if-you-are-a-community-member-seeking-a-review" >}}) and [wait]({{< ref "/contribute/getting-started/code-review#if-you-are-awaiting-a-review" >}}) for a [Focalboard core committer](https://github.com/mattermost/focalboard/blob/main/CONTRIBUTING.md#contributors) to review it. When in doubt, ask for help in the [Focalboard channel](https://community.mattermost.com/core/channels/focalboard) on our community server. If you are still stuck, please message Chen Lim ([@chenilim](https://github.com/chenilim) on GitHub).
+4. Submit your pull request for a [code review]({{< ref "/contribute/more-info/getting-started/code-review#if-you-are-a-community-member-seeking-a-review" >}}) and [wait]({{< ref "/contribute/more-info/getting-started/code-review#if-you-are-awaiting-a-review" >}}) for a [Focalboard core committer](https://github.com/mattermost/focalboard/blob/main/CONTRIBUTING.md#contributors) to review it. When in doubt, ask for help in the [Focalboard channel](https://community.mattermost.com/core/channels/focalboard) on our community server. If you are still stuck, please message Chen Lim ([@chenilim](https://github.com/chenilim) on GitHub).
 
 5. After a noteable bug fix or improvement is merged, submit a pull request to the [CHANGELOG](https://github.com/mattermost/focalboard/blob/main/CHANGELOG.md) under the next release section.
```

Comment:
@amyblais - This PR features new contributor guidance for our community. Given this context, is this step something we'd want community contributors taking action on? Or is this something that the Release Team owns and manages? Happy to leave as-is, modify for clarity, or remove altogether based on your feedback.

Reply:
Actually the Focalboard changelog is owned by the Boards team, so I'll defer to @wuwinson / @sbishel!